config/locales/cy.json

```diff
@@ -959,10 +959,9 @@
 		},
 		"programmes": {
 			"title": "Rhaglenni Ariannu",
-			"under10k": "Rhaglenni ariannu o dan £10,000",
-			"over10k": "Rhaglenni ariannu dros £10,000",
+			"under10k": "Gwobrau dan £10,000",
```

Comment:
Assuming we're happy with the general wording I need to confirm this robot translation before we go live.